packages/esm-form-engine-app/src/form-renderer/form-renderer.component.tsx

```diff
@@ -23,12 +27,24 @@ const FormRenderer: React.FC<FormRendererProps> = ({
   closeWorkspaceWithSavedChanges,
   encounterUuid,
   formUuid,
+  patient,
```

Comment:
We get the `patient` object as a prop for free. There's no need to make a call to `usePatient` to fetch it again.